pandas/io/excel.py

```diff
@@ -156,11 +159,18 @@ def get_writer(engine_name):
         Acceptable values are None or xlrd"""
 
 @Appender(excel_doc_common % read_excel_kwargs)
```

Comment:
sure, ``read_excel`` should also be able to take an ``ExcelFile`` object. Go ahead and add that on.
Yeah ``.parse`` was existing before ``read_excel``. go ahead and deprecate that as well (but obviously don't change the use for now). Point people to ``read_excel``. You'll need to change the docs as well.